mergin/client.py

```diff
@@ -372,6 +372,24 @@ def workspace_service(self, workspace_id):
 
         return response
 
+    def workspace_usage(self, workspace_id):
+        """
+        This Requests information about a workspace usage from /workspace/{id}/usage endpoint,
+        if such exists in self.url server.
+
+        Returns response from server as JSON dict or None if endpoint is not found
+        """
+
+        try:
+            response = self.get(f"/v1/workspace/{workspace_id}/usage")
+        except ClientError as e:
+            self.log.debug(f"Unable to query for /workspace/{workspace_id}/usage endpoint")
+            return
+
+        response = json.loads(response.read())
+
+        return response
```

Comment:
Just last wonder about this method. In other methods of mergin client class is ClientError raising back. I think it could be better to follow this behaviour.

```
try:
            self.post("/v1/workspace", params, {"Content-Type": "application/json"})
        except ClientError as e:
            e.extra = f"Workspace name: {workspace_name}"
            raise e
```

Reply:
Agreed, I am not even sure why I ended up diverging from the standard behaviour